get-started/the-stack.md

```diff
@@ -7,97 +7,157 @@ mapped_pages:
 
-{{es}} is the distributed search and analytics engine at the heart of the {{stack}}. It provides near real-time search and analytics for all types of data. Whether you have structured or unstructured text, numerical data, or geospatial data, {{es}} can efficiently store and index it in a way that supports fast searches. {{es}} provides a REST API that enables you to store data in {{es}} and retrieve it. The REST API also provides access to {{es}}'s search and analytics capabilities. [Learn more about {{es}}](/get-started/index.md).
+## Ingest [_ingest]
 
+Before you can search it, visualize it, and use it for insights, you must get your data into {{es}}.
+There are multiple methods for ingesting data.
+The best approach depends on the type of data and your specific use case.
+For example, you can collect and ship logs, metrics, and other types of data with {{agent}} or collect detailed performance information with {{product.apm}}.
+If you want to transform and enrich data before it's stored, you can use {{es}} ingest pipelines or {{ls}}.
 
-### Consume [_consume]
+Trying to decide which ingest components to use? Refer to [](/manage-data/ingest.md) and [](/manage-data/ingest/tools.md).
 
-Use {{kib}} to query and visualize the data that’s stored in {{es}}. Or, use the {{es}} clients to access data in {{es}} directly from common programming languages.
+### {{agent}} and {{integrations}}[stack-components-agent]
 
-#### {{kib}} [stack-components-kibana]
+{{agent}} is a single, unified way to add monitoring for logs, metrics, and other types of data to a host.
+It can also protect hosts from security threats, query data from operating systems, and forward data from remote services or hardware.
+Each agent has a single policy to which you can add [integrations](integration-docs://reference/index.md) for new data sources, security protections, and more.
+You can also use [{{agent}} processors](/reference/fleet/agent-processors.md) to sanitize or enrich your data.
 
-{{kib}} is the tool to harness your {{es}} data and to manage the {{stack}}. Use it to analyze and visualize the data that’s stored in {{es}}. {{kib}} is also the home for the Search, Observability and Security solutions. [Learn more about {{kib}}](/explore-analyze/index.md).
+To monitor the state of all your {{agents}}, manage agent policies, and upgrade {{agent}} binaries or integrations, refer to [Central management in {{fleet}}](/reference/fleet/index.md#central-management).
 
-#### {{es}} clients [stack-components-elasticsearch-clients]
+[Learn more about {{agent}}](/reference/fleet/index.md).
 
-The clients provide a convenient mechanism to manage API requests and responses to and from {{es}} from popular languages such as Java, Ruby, Go, Python, and others. Both official and community contributed clients are available. [Learn more about the {{es}} clients](/reference/elasticsearch-clients/index.md).
+### {{product.apm}} [stack-components-apm]
```

Comment:
problem for later: is APM considered to be a stack component since it's observability only? 😵 maybe in future we can call that out

Reply:
Thanks, added a link to this for follow-up